models/prices/prices_usd_latest.sql

```diff
@@ -4,24 +4,17 @@
         post_hook='{{ expose_spells(\'["ethereum", "solana", "arbitrum", "gnosis", "optimism", "bnb", "avalanche_c"]\',
                                     "sector",
                                     "prices",
-                                    \'["hildobby"]\') }}'
+                                    \'["hildobby", "0xRob"]\') }}'
         )
 }}
 
-SELECT pu.blockchain
+SELECT
+  pu.blockchain
 , pu.contract_address
 , pu.decimals
-, pu.minute
-, pu.price
 , pu.symbol
-FROM (
-    SELECT blockchain
-    , contract_address
-    , MAX(minute) AS latest
-    FROM {{ source('prices', 'usd') }}
-    GROUP BY blockchain, contract_address
-    ) latest
-LEFT JOIN {{ source('prices', 'usd') }} pu ON pu.blockchain=latest.blockchain
-    AND pu.contract_address=latest.contract_address
-    AND pu.minute=latest.latest
-
+, max(pu.minute) as minute
+, max_by(pu.price, pu.minute) as price
+FROM {{ source('prices', 'usd') }} pu
+WHERE minute > now() - interval 7 day
```

Comment:
The issue we would have with this for now is DuneSQL compatibility

Reply:
Exactly, this would only work if this was materialized (which I'm guessing we'd like to avoid considering the nature of the info it holds)